src/node.ts

```diff
@@ -97,10 +104,15 @@ export class EthereumNode extends constructs.Construct {
     // will be populated with defaults when passed to the node constructor
     this.network = props.network ?? Network.MAINNET;
     this.instanceType = props.instanceType ?? InstanceType.BURSTABLE3_LARGE;
-    this.availabilityZone = props.availabilityZone ?? `${region}a`;
 
-    utilities.validateRegion(region);
-    utilities.validateAvailabilityZone(region, this.availabilityZone);
+    // If no availability zone is provided, use the first in the region.
+    if (regionInEnvironment) {
+      utilities.validateRegion(region);
+      this.availabilityZone = props.availabilityZone ?? `${region}a`;
```

Comment:
Now that we have `region` as an output parameter we need to set it with `this.region = region;`

Reply:
Resolved

Reply:
Yes, I just missed it!